common/p_setup.cpp

```diff
@@ -97,6 +97,8 @@ line_t* 		lines;
 int 			numsides;
 side_t* 		sides;
 
+int*			originalLightLevels; // Needed for map resets
```

Comment:
Is there a reason why you used an `int*` and `new[]` here instead of ` std::vector`?  Also, I don't see a `delete[]` anywhere.

Reply:
I redid this to use std::vector instead.